Cargo.toml

```diff
@@ -150,7 +154,10 @@ native_tls_backend = ["reqwest/default-tls", "tungstenite/tls"]
 model = ["builder", "http"]
 standard_framework = ["framework", "uwl", "command_attr"]
 utils = ["base64"]
-voice = ["byteorder", "gateway", "audiopus", "rand", "sodiumoxide"]
+voice = ["byteorder", "gateway", "audiopus", "rand"]
+voice-secretbox = ["voice", "secretbox"]
```

Comment:
We use underscore for features.
Furthermore, orientate yourself on our naming for the HTTP-backend.
Use "sodiumoxide_backend"/"secretbox_backend".

We will have to see how we are going to rename this to avoid confusion with now four features containing backend. This might be "voice/sodiumoxide" etc., but that is out of the scope here.